src/FakeItEasy/ExceptionThrowerConfigurationExtensions.cs

```diff
@@ -130,6 +134,11 @@ public static IAfterCallSpecifiedConfiguration Throws<T1>(this IExceptionThrower
                 });
         }
 
+#if DISABLED
+
+        // This method already has a generic parameter, and it can't be inferred; if we add another, we make it unusable,
+        // especially since the new one is something unintelligible like IAnyCallConfigurationWithReturnTypeSpecificied<Foo>...
+

```

Comment:
> I've always seen these extension methods as coding convenience, not necessity.

Well, they *are* a convenience, but without them, the API becomes quite inconvenient...

> Could we make this not be an extension method? e.g., on RuleBuilder (and other leaf classes) have a method like

Yes, but we don't expose the classes directly, only the interfaces... so the method wouldn't be available (unless we make it part of the interface, but that would make it harder to implement)

However, your comment made me realize that it should be possible to write more specific extension methods that wouldn't require the extra type parameter; looking into it now.